portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/portalconfiguration/systemmanagement/SystemSettings.testcase

```diff
@@ -19,6 +19,31 @@ definition {
 		}
 	}
 
+	@description = "LPS-194725 - This test asserts that the hyperlink is not embedded within the cancel button."
+	@priority = 3
+	test AssertHyperlinkIsNotRenderedInAButton {
+		task ("Given signed into portal as admin") {
+			Navigator.openURL();
+		}
+
+		task ("When navigate to Bundle Blacklist edit configuration page ") {
+			Navigator.openSpecificURL(url = "${portalURL}/group/control_panel/manage?p_p_id=com_liferay_configuration_admin_web_portlet_SystemSettingsPortlet&p_p_lifecycle=0&p_p_state=maximized&p_p_mode=view&_com_liferay_configuration_admin_web_portlet_SystemSettingsPortlet_factoryPid=com.liferay.portal.bundle.blacklist.internal.configuration.BundleBlacklistConfiguration&_com_liferay_configuration_admin_web_portlet_SystemSettingsPortlet_mvcRenderCommandName=%2Fconfiguration_admin%2Fedit_configuration&_com_liferay_configuration_admin_web_portlet_SystemSettingsPortlet_pid=com.liferay.portal.bundle.blacklist.internal.configuration.BundleBlacklistConfiguration");
```

Comment:
This specific URL worked fine for me when I ran the test locally, it looks weird now... Let's see if it works fine on CI as well. 🤔

Reply:
Every time I update the branch, the URL changes... It might be worth to reach to that part by the UI instead of using `Navigator.openSpecificURL`

![image](https://github.com/liferay-peds/liferay-portal/assets/63858122/bbdef8a7-972b-49e3-a6bf-0846716fb819)